example/spec/parallel/createReconciliationReport/CreateReconciliationReportSpec.js

```diff
@@ -428,7 +427,8 @@ describe('When there are granule differences and granule reconciliation is run',
     if (beforeAllFailed) fail(beforeAllFailed);
   });
 
-  describe('Create an Inventory Reconciliation Report to monitor inventory discrepancies', () => {
+  // TODO: fix tests in CUMULUS-3806 when CreateReconciliationReport lambda is changed to query postgres
```

Comment:
TODOs aren't really sufficient to ensure this gets done right?   We should probably add comments for all of these to the tickets AC or comments to ensure they're *specifically* addressed before merging to the main branch.

Reply:
added comment to ticket CUMULUS-3806

Reply:
👍🏻  I looked at the other ticket in this CL and I think they're all covered.